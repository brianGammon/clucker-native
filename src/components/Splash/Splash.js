import * as React from 'react';
import { ActivityIndicator, View } from 'react-native';

import styles from './styles';

const Splash = () => (
  <View style={styles.container}>
    <ActivityIndicator size="large" />
  </View>
);

export default Splash;
