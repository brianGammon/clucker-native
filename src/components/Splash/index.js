import * as React from 'react';
import { ActivityIndicator, StatusBar, View } from 'react-native';

import styles from './styles';

export default () => (
  <View style={styles.container}>
    <ActivityIndicator size="large" />
    <StatusBar barStyle="default" />
  </View>
);
