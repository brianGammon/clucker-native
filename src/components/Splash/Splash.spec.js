import React from 'react';
import Splash from '.';

describe('Splash component:', () => {
  test('Should render correctly', () => {
    const wrapper = shallow(<Splash />);
    expect(wrapper).toMatchSnapshot();
  });
});
