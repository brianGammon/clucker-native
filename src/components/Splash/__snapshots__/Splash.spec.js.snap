// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash component: Should render correctly 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Component
    size="large"
  />
</Component>
`;
